en/08_Changelogs/5.4.0.md

```diff
@@ -127,37 +127,43 @@ The changes include:
 - [`SSViewer::templates()`](api:SilverStripe\View\SSViewer::templates()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`SSViewer::setTemplateFile()`](api:SilverStripe\View\SSViewer::setTemplateFile()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`SSViewer::get_base_tag()`](api:SilverStripe\View\SSViewer::get_base_tag()) has been deprecated. Use [`SSViewer::getBaseTag()`](api:SilverStripe\View\SSViewer::getBaseTag()) instead.
-- [`SSViewer_DataPresenter`](api:SilverStripe\View\SSViewer_DataPresenter) has been deprecated. It will be merged back into [`SSViewer_Scope`](api:SilverStripe\View\SSViewer_Scope).
-- [`SSViewer_FromString`](api:SilverStripe\View\SSViewer_FromString) has been deprecated. It will be replaced with `SilverStripe\View\SSTemplateEngine::renderString()`.
+- [`SSViewer_DataPresenter`](api:SilverStripe\View\SSViewer_DataPresenter) has been deprecated. It will be merged into [`SSViewer_Scope`](api:SilverStripe\TemplateEngine\ScopeManager).
+- [`SSViewer_FromString`](api:SilverStripe\View\SSViewer_FromString) has been deprecated. It will be replaced with `SilverStripe\TemplateEngine\SSTemplateEngine::renderString()`.
 - [`SSViewer_Scope::getItem()`](api:SilverStripe\View\SSViewer_Scope::getItem()) has been deprecated. Use [`SSViewer_Scope::getCurrentItem()`](api:SilverStripe\View\SSViewer_Scope::getCurrentItem()) instead.
-- [`SSViewer_Scope::obj()`](api:SilverStripe\View\SSViewer_Scope::obj()) has been deprecated. It will be renamed `scopeToIntermediateValue()`.
+- [`SSViewer_Scope::obj()`](api:SilverStripe\View\SSViewer_Scope::obj()) has been deprecated. It will be renamed to `scopeToIntermediateValue()`.
+- [`SSViewer_Scope`](api:SilverStripe\View\SSViewer_Scope) has been deprecated. It will be renamed to `SilverStripe\TemplateEngine\ScopeManager`.
 - [`SSViewer::execute_template()`](api:SilverStripe\View\SSViewer::execute_template()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`SSViewer::execute_string()`](api:SilverStripe\View\SSViewer::execute_string()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`ThemeResourceLoader::findTemplate()`](api:SilverStripe\View\ThemeResourceLoader::findTemplate()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`FlushMiddleware`](api:SilverStripe\Control\Middleware\FlushMiddleware) has been deprecated. It will be replaced with flushing inside the Kernel directly.
-- The [`LeftAndMain.tree_class`](api:SilverStripe\Admin\LeftAndMain->tree_class) configuration property has been deprecated. It will be renamed `model_class`.
+- The [`LeftAndMain.tree_class`](api:SilverStripe\Admin\LeftAndMain->tree_class) configuration property has been deprecated. It will be renamed to `model_class`.
```

Comment:
Not related, but all the other rename notices say "renamed _to_" so I'm updating this to be consistent.